src/toggle.jsx

```diff
@@ -13,7 +13,7 @@ var Toggle = React.createClass({
   },
 
   propTypes: {
-    elementStyle: React.PropTypes.object,
+    toggleElementBoxStyle: React.PropTypes.object,
```

Comment:
This prop rename will introduce a breaking change. Can we keep as is for now pls?